pngcheck: add page (#9015)
tldr-pages/tldr: pages/linux/pngcheck.md

# pngcheck

> > Useful forensics tool for validating the integrity of PNG based (.png|.jng|.mng) image files.

Reviewer: ```suggestion
> Forensics tool for validating the integrity of PNG based (`.png`, `.jng`, `.mng`) image files.
```

> > Can also extract embedded images and text from the file.

Reviewer: ```suggestion
> Can also extract embedded images and text from a file.
```

> More information: <http://www.libpng.org/pub/png/apps/pngcheck.html>.

- Verify the integrity of an image file:

> `pngcheck {{image}}`

Reviewer: Are we talking about images like the created with the `dd` command? Perhaps this better
```suggestion
`pngcheck {{file.img}}`
```

Author: Thank you for the feedback.  I hadn't thought about the ambiguity of the term.  It's actually for PNG image files.  I have updated the description to include (.png | .jng | .mng) image types and changed file.png in the examples.

> `pngcheck {{file.png}}`

Reviewer: We use `path/to/file` as placeholder
```suggestion
`pngcheck {{path/to/file.png}}`
```

- Check the file with [v]erbose and [c]olorized output:

> `pngcheck -vf {{file.png}}`

Reviewer: ```suggestion
`pngcheck -vf {{path/to/file.png}}`
```

> `pngcheck -vf {{path/to/file.png}}`

Reviewer 2: ![\[c\]](https://user-images.githubusercontent.com/35269695/196065072-4b89b0a8-7633-4604-8b7c-87a09d1378f9.png)
![-vf](https://user-images.githubusercontent.com/35269695/196065095-f55bfff9-716b-40d6-b46e-058b8ce2fdad.png)

Am I missing something?

Author: Fixed in commit 
e9e28b0

Thank you

> - Display contents of [t]ext chunks and [s]earch for PNGs within the file:

Reviewer: ```suggestion
- Display contents of [t]ext chunks and [s]earch for PNGs within a specific file:
```

`pngcheck -ts {{file.png}}`

> - Search for, and e[x]tract embedded PNGs within the file:

Reviewer: ```suggestion
- Search for, and e[x]tract embedded PNGs within a specific file:
```

> `pngcheck -x {{file.png}}`

Reviewer: ```suggestion
`pngcheck -x {{path/to/file.png}}`
```
